torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -1514,6 +1514,36 @@ class IrParser {
       }
     }
 
+    {
+      auto ptr_op = getOperatorForLiteral(
+          "aten::index_select(Tensor self, int dim, Tensor index) -> Tensor");
```

Comment:
I think enablement in TorchScript by default is a big change because it means direct exposure to the user. I don't think we have done sufficient validation to do so. However, we can add an environment flag in https://github.com/csarofeen/pytorch/blob/c9f8c1de5f3bd6ac0a0cce50569410abe2dfc209/torch/csrc/jit/codegen/cuda/utils.h#L89
So that it is enabled only when a user manually does so. This way, we can both continue our development with this flag on, and start using it in PyG by manually enabling it, and at the same time avoid expose to all users by default.